### DISCUSSION TITLE
Set the size of the emulated console

### DISCUSSION THREAD

**Discussion by arogl** [gitextensions/gitextensions#9592] [Q&A]
Does the emulated console allow you to set the default size of the console windows?

**Comment by RussKie** [comment]
Can you please elaborate?

**Reply by arogl** [reply]
The default console for bash (mintty)/cmd, you can set the number of rows and columns for the default size of the window that opens.

I noticed that I could not see such an option for the emulated console in git extensions, I also noticed that when I resized an emulated console for a git operation, the size of the window was remembered. I do not know if this is persistent across git extension restarts.

Hoping for something similar with the emulated console.

![image](https://user-images.githubusercontent.com/1115472/134842288-cfa994fc-c938-4c6b-b80d-7d8630a4da62.png)
